Configuration/Development.php

```diff
@@ -28,6 +28,10 @@
 ];
 $GLOBALS['TYPO3_CONF_VARS']['LOG'] = array_replace_recursive($GLOBALS['TYPO3_CONF_VARS']['LOG'], $logWriterConf);
 
+// Disable password policies (since it's annoying in dev contexts)
+$GLOBALS['TYPO3_CONF_VARS']['BE']['passwordPolicy'] = '';
+$GLOBALS['TYPO3_CONF_VARS']['FE']['passwordPolicy'] = '';
```

Comment:
better place this below in the `return` array instead of separated, so that its together with the remaining `BE` and `FE` settings.